src/Spice86.Shared/Rgb.cs

```diff
@@ -25,4 +25,12 @@ public class Rgb {
     public override string ToString() {
         return System.Text.Json.JsonSerializer.Serialize(this);
     }
+
+    public static implicit operator uint(Rgb v) {
+        return ToUint(v);
+    }
+
+    public static uint ToUint(Rgb v) {
+        return (uint)(v.R << 16 | v.G << 8 | v.B);
```

Comment:
Maybe create a toRgb() method in the class and call it from here and from toArgb() ?

Reply:
Done.